content/archive/A Simple Honcho Primer.md

```
@@ -5,6 +5,10 @@ tags:
   - blog
   - honcho
 ---
+> [!custom]  WELCOME TO THE PLASTIC ARCHIVE WIP
+> This post has been archived because it's legacy content that 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)